inference-engine/thirdparty/clDNN/tests/test_cases/fusings_gpu_test.cpp

```diff
@@ -2989,94 +3032,233 @@ TEST_P(activation_quantize_i8, basic) {
-                      quantize("quant", "act", "in_low", "in_high", "out_low", "out_high", 255, data_types::i8),
-                      reorder("reorder_bfyx", "quant", format::bfyx, data_types::f32));
+                      data("out_low", get_mem(get_single_element_layout(p), 0)),
+                      data("out_high", get_mem(get_single_element_layout(p), 255)),
+                      scale("scale", "act", "scale_data"),
+                      activation("act2", "scale", activation_func::exp),
+                      quantize("quant", "act2", "in_low", "in_high", "out_low", "out_high", 255, data_types::u8),
+                      reorder("reorder_bfyx", "quant", p.default_format, data_types::f32));
 
     tolerance = 1.f;
     execute(p);
 }
 
 INSTANTIATE_TEST_CASE_P(
     fusings_gpu,
-    activation_opt_quantize_i8,
+    activation_quantize_scale_activation,
+    ::testing::ValuesIn(std::vector<activation_test_params>{
+        // InputDataType = FP32
+        activation_test_params{CASE_ACTIVATION_F32_0, 2, 5, "activation_opt"},
+        activation_test_params{CASE_ACTIVATION_F32_1, 2, 5, "activation_opt"},
+        activation_test_params{CASE_ACTIVATION_F32_2, 2, 5, "activation_opt"},
+        activation_test_params{CASE_ACTIVATION_F32_3, 2, 5, "activation_opt"},
+        activation_test_params{CASE_ACTIVATION_3D_F32_0, 2, 5, "activation_opt"},
+        activation_test_params{CASE_ACTIVATION_3D_F32_1, 2, 5, "activation_opt"},
+        activation_test_params{CASE_ACTIVATION_3D_F32_2, 2, 5, "activation_opt"},
+        activation_test_params{CASE_ACTIVATION_3D_F32_3, 2, 5, "activation_opt"},
+
+        activation_test_params{CASE_ACTIVATION_F32_0, 2, 5, "activation_ref"},
+        activation_test_params{CASE_ACTIVATION_F32_1, 2, 5, "activation_ref"},
+        activation_test_params{CASE_ACTIVATION_F32_2, 2, 5, "activation_ref"},
+        activation_test_params{CASE_ACTIVATION_F32_3, 2, 5, "activation_ref"},
+        activation_test_params{CASE_ACTIVATION_F32_4, 2, 5, "activation_ref"},
+        activation_test_params{CASE_ACTIVATION_F32_5, 2, 5, "activation_ref"},
+        activation_test_params{CASE_ACTIVATION_F32_6, 2, 5, "activation_ref"},
+        activation_test_params{CASE_ACTIVATION_F32_7, 2, 5, "activation_ref"},
+        activation_test_params{CASE_ACTIVATION_3D_F32_0, 2, 5, "activation_ref"},
+        activation_test_params{CASE_ACTIVATION_3D_F32_1, 2, 5, "activation_ref"},
+        activation_test_params{CASE_ACTIVATION_3D_F32_2, 2, 5, "activation_ref"},
+        activation_test_params{CASE_ACTIVATION_3D_F32_3, 2, 5, "activation_ref"},
+        activation_test_params{CASE_ACTIVATION_3D_F32_4, 2, 5, "activation_ref"},
+    }), );
+
+INSTANTIATE_TEST_CASE_P(
+    DISABLED_fusings_gpu,
+    activation_quantize_scale_activation,
+    ::testing::ValuesIn(std::vector<activation_test_params>{
+        activation_test_params{CASE_ACTIVATION_F32_8, 2, 3, "activation_ref"},     // FIXME - accuracy bug
```

Comment:
`// FIXME - accuracy bug` is it some new accuracy bug?

Reply:
Yes, it's connected with fusing formats in disabled tests to activation_ref kernel.